src/revlanguage/datatypes/phylogenetics/trees/RlTreeVector.cpp

```diff
@@ -0,0 +1,114 @@
+#include "RlMultiValueEvent.h"
```

Comment:
So RlTreeVector.cpp only includes code for MultiValueEvent?

It looks like this file was copied from RlMultiValueEvent.cpp so that it could be modifed, but was never modified.

Reply:
Good point. Don't remember what the purpose was but looks like we can delete it.

Reply:
That one has been removed.